### DISCUSSION TITLE
zh-style-guide 的使用者列表（完善中）

### DISCUSSION THREAD

**Discussion by yikeke** [yikeke/zh-style-guide#26] [Show and tell]
如果你的公司也 fork 了该风格指南，欢迎在下方留言分享 `<公司名称>：<中文文档官网链接>，<风格指南链接>`，鼓励大家一起学习交流～

目前已知的使用者：

- PingCAP：[中文文档官网](https://docs.pingcap.com/zh/tidb/dev)，[PingCAP 中文技术文档风格指南](https://github.com/pingcap/docs-cn/blob/master/resources/pingcap-style-guide-zh.pdf)
- 待补充……

**Comment by Chaisson-Li** [comment]
docboon writing style
